libs/styles/src/lib/scss/components/_notice.scss

```diff
@@ -0,0 +1,58 @@
+@mixin cvi-notice($base: &) {
+  width: 100%;
+  #{$base}__container {
+    border-radius: var(--cvi-radius-s);
+    display: flex;
+    color: get-color(black-coral-12);
+    fill: get-color(black-coral-12);
+    font-size: 14px;
+  }
+
+  #{$base}__content {
+    align-items: center;
+    padding: var(--cvi-spacing-valga);
+    width: 100%;
+  }
+
+  #{$base}__icon-container {
+    display: flex;
+    align-items: center;
+    justify-content: center;
+    padding: 10px;
+  }
+
+  #{$base}--severity-success {
+    border: 1px solid var(--cvi-color-black-coral-2);
+
+    & #{$base}__icon-container {
+      background-color: var(--cvi-color-sea-green-2);
+      fill: var(--cvi-color-sea-green-10);
+    }
+  }
+
+  #{$base}--severity-info {
+    border: 1px solid var(--cvi-color-black-coral-2);
+    & #{$base}__icon-container {
+      background-color: var(--cvi-color-sapphire-blue-2);
+      fill: var(--cvi-color-sapphire-blue-10);
+    }
+  }
+
+  #{$base}--severity-error {
+    border: 1px solid var(--cvi-color-jasper-10);
+
+    & #{$base}__icon-container {
+      background-color: var(--cvi-color-jasper-2);
+      fill: var(--cvi-color-jasper-10);
+    }
+  }
+
+  #{$base}--dark-title {
+    color: get-color(black);
+    font-size: 16px;
+
+    &.underlined {
```

Comment:
The title seems to be always un underlined state, perhaps makes sense to remove the separate modifier class and just incorporate the style into `#{$base}__dark-title`.

Reply:
There are some instances where the title is without underline (date field)

Reply:
There are some instances where the title is not underlined (date)

Reply:
It might be a bit confusing that a date uses a class called `__title`. However, this is not a big problem.